include/megaapi.h

```diff
@@ -14716,10 +14862,79 @@ class MegaApi
          *
          * When onRequestFinish is called with MegaError::API_OK, the SDK is guaranteed to be
          * up to date (as for the time this function is called).
+         * @brief Send a verification code txt to the supplied phone number
+         *
+         * Sends a 6 digit code to the user's phone.  The number is supplied in this function call.
+         * The code is txted to the user.  Once the user has it, they can type it into the app
+         * and the call checkSMSVerificationCode() can be used to validate the user did receive the txt,
+         * so that really is their phone number.
+         *
+         * The frequency with which this call can be used is very limited (once or twice per
+         * 24 hour period, at the time of writing this), so it's important to get the 
+         * number right on the first try.  The result will be API_ETEMPUNAVAIL if it has
+         * been tried too frequently.
+         *
+         * Make sure to test the result of smsAllowedState() before calling this function.
          *
          * @param listener MegaRequestListener to track this request
          */
-        void catchup(MegaRequestListener *listener = NULL);
+        void sendSMSVerificationCode(const char* phoneNumber, MegaRequestListener *listener = NULL, bool reverifying_whitelisted = false);
+ 
+        /**
+         * @brief Check a verification code that the user should have received via txt
+         *
+         * This function validates that the user received the txt sent by sendSMSVerificationCode().
+         * If the user supplied the right code, the result will be API_OK.
+         *
```

Comment:
The error code comes direct from the JSON reply, so these are server controlled, and the list of possible error may change or expand in future - if we list the current ones, it may lead 3rd party developers to think that the list will not change in future?

Reply:
As it turns out, these are documented in the wiki - I've added them to the header documentation.  cheers